packages/components/src/components/checkbox-wrapper/checkbox-wrapper-styles.ts

```diff
@@ -3,24 +3,36 @@ import { getCheckboxRadioJssStyle } from '../../styles/checkbox-radio-styles';
 import type { FormState } from '../../utils/form/form-state';
 import { getCss, mergeDeep } from '../../utils';
 import { getInlineSVGBackgroundImage } from '../../utils/svg/getInlineSVGBackgroundImage';
-import { addImportantToEachRule, getThemedColors } from '../../styles';
-import { borderRadiusMedium, borderRadiusSmall } from '@porsche-design-system/utilities-v2';
+import { addImportantToEachRule, getThemedColors, getTransition } from '../../styles';
+import { borderRadiusMedium, borderRadiusSmall, fontFamily, fontLineHeight } from '@porsche-design-system/utilities-v2';
 
 export const getComponentCss = (
   hideLabel: BreakpointCustomizable<boolean>,
   state: FormState,
   isDisabled: boolean,
+  isLoading: boolean,
+  isChecked: boolean,
   theme: Theme
 ): string => {
+  const themedColors = getThemedColors(theme);
   const checkedIconColor = getThemedColors(theme === 'light' ? 'dark' : 'light').primaryColor.replace(/#/g, '%23');
-  const indeterminateIconColor = getThemedColors(theme).primaryColor.replace(/#/g, '%23');
+  const indeterminateIconColor = themedColors.primaryColor.replace(/#/g, '%23');
```

Comment:
I guess `primaryColor` could be destructured now next to `disabledColor` in the upcoming line.

Reply:
fixed it